editor/src/components/canvas/canvas-utils-scene.spec.browser.ts

```diff
@@ -4,7 +4,7 @@ import {
   TestScenePath,
   makeTestProjectCodeWithSnippet,
   getTestParseSuccess,
-} from './ui-jsx-test-utils' // IMPORTANT - THIS IMPORT MUST ALWAYS COME FIRST
+} from './ui-jsx.test-utils' // IMPORTANT - THIS IMPORT MUST ALWAYS COME FIRST
```

Comment:
Why is this renamed?

Reply:
see the description

> Renaming *-test-utils.ts files to *.test-utils.ts to make it very obvious that these are special files (not to be imported from production code!)

Reply:
I've made a dependency cruiser rule that fails if a non-spec file imports from a .test-utils file. But I didn't want to make the rule depend on `-test-utils`, I felt like that might be too unexpected. So I renamed these to .test-utils.ts to mark that something special is going on